### ISSUE TITLE
fails to build on Debian Jessie

### ISSUE THREAD

**Issue by mestia** [rmagick-temp/rmagick#131]
dpkg -l | grep magick | awk '{print $2,$3}'
imagemagick-common 8:6.8.9.9-5
libmagick++-6-headers 8:6.8.9.9-5
libmagick++-6.q16-5:amd64 8:6.8.9.9-5
libmagick++-6.q16-dev:amd64 8:6.8.9.9-5
libmagick++-dev 8:6.8.9.9-5
libmagickcore-6-arch-config:amd64 8:6.8.9.9-5
libmagickcore-6-headers 8:6.8.9.9-5
libmagickcore-6.q16-2:amd64 8:6.8.9.9-5
libmagickcore-6.q16-2-extra:amd64 8:6.8.9.9-5
libmagickcore-6.q16-dev:amd64 8:6.8.9.9-5
libmagickwand-6-headers 8:6.8.9.9-5
libmagickwand-6.q16-2:amd64 8:6.8.9.9-5
libmagickwand-6.q16-dev:amd64 8:6.8.9.9-5
libmagickwand-dev 8:6.8.9.9-5

Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

```
/usr/bin/ruby2.1 extconf.rb 
```

checking for gcc... yes
checking for Magick-config... no
checking for pkg-config... yes
checking for outdated ImageMagick version (<= 6.4.9)... **\* extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
        --with-opt-dir
        --without-opt-dir
        --with-opt-include
        --without-opt-include=${opt-dir}/include
        --with-opt-lib
        --without-opt-lib=${opt-dir}/lib
        --with-make-prog
        --without-make-prog
        --srcdir=.
        --curdir
        --ruby=/usr/bin/ruby2.1
extconf.rb:79:in ``': No such file or directory - convert (Errno::ENOENT)
        from extconf.rb:79:in`block in configure_compile_options'
        from /usr/lib/ruby/2.1.0/mkmf.rb:918:in `block in checking_for'
        from /usr/lib/ruby/2.1.0/mkmf.rb:351:in`block (2 levels) in postpone'
        from /usr/lib/ruby/2.1.0/mkmf.rb:321:in `open'
        from /usr/lib/ruby/2.1.0/mkmf.rb:351:in`block in postpone'
        from /usr/lib/ruby/2.1.0/mkmf.rb:321:in `open'
        from /usr/lib/ruby/2.1.0/mkmf.rb:347:in`postpone'
        from /usr/lib/ruby/2.1.0/mkmf.rb:917:in `checking_for'
        from extconf.rb:76:in`configure_compile_options'
        from extconf.rb:15:in `initialize'
        from extconf.rb:473:in`new'
        from extconf.rb:473:in `<main>'

extconf failed, exit code 1

I could build the previous verions of rmagick successfully.

**Comment by running0x** [comment]
Did you figure this out? I'm having this today too.

**Comment by mestia** [comment]
I just copied the old rmagick I had in the old version of redmine and that's all.

The strange thing is that the rmagick is looking for Magick-config which is present in the following packages:

apt-file search Magick-config
graphicsmagick-libmagick-dev-compat: /usr/bin/Magick-config
graphicsmagick-libmagick-dev-compat: /usr/share/man/man1/Magick-config.1.gz
libgraphicsmagick1-dev: /usr/bin/GraphicsMagick-config
libgraphicsmagick1-dev: /usr/share/man/man1/GraphicsMagick-config.1.gz
libmagickcore-6.q16-dev: /usr/lib/x86_64-linux-gnu/ImageMagick-6.8.9/bin-Q16/Magick-config

libmagickcore-6.q16-dev is installed on my system, but the configure? can not find it.

I've also tried to substitute ImageMagick with GraphicsMagick but this didn't help much, there was a problem in some other missing dependencies for rmagick.
